apisix/plugins/echo.lua

```diff
@@ -31,6 +34,11 @@ local schema = {
         after_body = {
             description = "body after the modification of filter phase.",
             type = "string"
+        },
+        headers = {
+            description = "new headers for repsonse",
+            type = "object",
+            minProperties = 1,
```

Comment:
can we only require the `headers`?

Reply:
if the answer is YES, then we need to update the process: https://github.com/apache/incubator-apisix/pull/1708/files#diff-32537bd33cf9a44d9dd277b87d1ce5f9R82

Reply:
I prefer to keep them since it is possible for someone to change the body of the response in the body filter part phase from what the user inputs. It is easy to understand that way.